rskj-core/src/main/java/org/ethereum/listener/GasPriceTracker.java

```diff
@@ -61,30 +75,39 @@ public void onBlock(Block block, List<TransactionReceipt> receipts) {
         logger.trace("End onBlock");
     }
 
-    public void onTransaction(Transaction tx) {
+    private void onTransaction(Transaction tx) {
```

Comment:
Empty blocks or unusued block space should reduce the suggested gas price, which is not happening with this formula. Probably an idea to track in the backlog, nothing actually related to this task